test/_function_configs/error/s3_codeentry/function.yaml

```diff
@@ -0,0 +1,30 @@
+metadata:
```

Comment:
Please remove irrelevant fields to this test scenario
That will give us more confident while would be able to ensure default values are inserted correctly
and make this `function.yaml` more readable when trying to figure what values are "cause" the error

also, replace "sdfsd" values with "Dummy-X" (e.g. s3Bucket) so logs would be referenced correctly to the `function.yaml` values

From a quick look you can omit
`meta.labels`
`spec.triggers,env,disable,loggerSinks
`spec.build.runtimeAttributes,noCache,offline,dependencies,image,mode`

Reply:
Sure thing